.dockerignore

```diff
@@ -0,0 +1,2 @@
+frontend
```

Comment:
...in order to build the docker image faster (it sends build context to Docker daemon and takes a long time if it sends unneeded things)